public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Calculator</title>
   </head>
```

Comment:
- [ ] You have done a great job removing some of the boilerplate files, however, there're still some files that aren't removed kindly remove them.

![image](https://user-images.githubusercontent.com/59416136/170361682-6b467023-c0d9-4d47-b8da-f1a76514086e.png)